enable_rangefinder: [False, 'Add a laser rangefinder (Garmin) pointing down', [f450, f550, t650]]
enable_rangefinder_up: [False, 'Add a laser rangefinder (Garmin) pointing up', [f450, f550, t650]]
enable_teraranger: [False, 'Add a laser rangefinder (Teraranger) to the vehicle', [f450, f550, t650]]
enable_ground_truth: [False, 'Enable topic with ground truth odometry', [f450, f550, t650]]
enable_mobius_camera_front: [False, 'Add mobius camera to the vehicle [1280x720 30hz], pointed to the front', [f450, f550, t650]]
enable_mobius_camera_down: [False, 'Add mobius camera to the vehicle [1280x720 30hz], pointed to the ground', [f450, f550]]
enable_mobius_camera_back_left: [False, 'Add mobius camera to the vehicle [1280x720 30hz], pointed to the back left', [f450, f550]]
enable_mobius_camera_back_right: [False, 'Add mobius camera to the vehicle [1280x720 30hz], pointed to the back right', [f450, f550]]
enable_realsense_front_pitched: [False, 'Add Intel Realsense D435 depth camera to the vehicle [1280x720 30hz], pointed to the front pitched down by 10 up to 45 deg. Depends on the type of vehicle', [f450, f550]]
enable_realsense_down: [False, 'Add Intel Realsense D435 depth camera to the vehicle [1280x720 30hz], pointed down', [f450, f550]]
enable_realsense_top: [False, 'Add Intel Realsense D435 depth camera to the vehicle [1280x720 30hz], pointed forward placed on the aluminum frame', [f550]]
enable_realsense_front: [False, 'Add Intel Realsense D435 depth camera to the vehicle [1280x720 30hz], pointed forward placed on the front holder between the legs', [f450, f550, t650]]
enable_realsense_up: [False, 'Add Intel Realsense D435 depth camera to the vehicle [1280x720 30hz], pointed upwards', [f450, f550, t650]]
use_realistic_realsense: [False, 'Enable a more realistic simulation of the Realsense D435 depth image (only takes effect if some Realsense is enabled)', [f450, f550, t650]]
enable_whycon_box: [False, 'Add a Whycon box for relative localization', [f450, f550]]
enable_bluefox_camera: [False, 'Add bluefox camera to the vehicle [752x480 50hz]', [f450, f550, t650]]
enable_bluefox_camera_reverse: [False, 'Add bluefox camera to the vehicle [752x480 50hz], rotated by 180 deg', [f450, f550, t650]]
enable_fisheye_camera: [False, 'Add fisheye camera to the vehicle [752x480 180deg 45hz]', [f450]]
enable_magnetic_gripper: [False, 'Add magnetic gripper', [f450, f550, t650]]
enable_scanse_sweep: [False, 'Add Scanse Sweep laser scanner to the vehicle', [f450, f550, t650]]
enable_rplidar: [False, 'Add the RPlidar A3 laser scanner to the vehicle', [f450, f550, t650]]
enable_pendulum: [False, 'Add pendulum to the vehicle', [f450, f550, t650]]
enable_teraranger_tower_evo: [False, 'Add the Teraranger Tower Evo laser scanner to the vehicle', [f450, f550, t650]]
enable_timepix: [False, 'Add Timepix radiation detector to the vehicle', [f450, f550, t650]]
enable_thermal_camera: [False, 'Add thermal camera to the vehicle', [t650]]
enable_velodyne: [False, 'Add Velodyne PUCK Lite laser scanner to the vehicle', [f450, f550, t650]]
enable_ouster: [False, 'Add Ouster laser scanner to the vehicle', [f450, f550, t650]]
ouster_model: [None, 'Choose the Ouster model. Options: OS0-32, OS0-64, OS0-128, OS1-16, OS1-32G1, OS1-32, OS1-64, OS1-128, OS2-32, OS2-64, OS2-128', [f450, f550, t650]]
use_gpu_ray: [False, 'Laser ray casting will be handled by GPU shaders', [f450, f550, t650]]
enable_light: [False, 'Add light to the vehicle', [f450, f550, t650]]
enable_servo_camera: [False, 'Add camera on virtual servo to the vehicle', [f450, f550, t650]]
enable_ball_holder: [False, 'Add ball holder to the vehicle', [f550, t650]]
enable_water_gun: [False, 'Add water gun for fire fighting', [t650]]
enable_parachute: [False, 'Add parachute for emergency flight termination', [t650]]
wall_challenge: [False, 'Configuration for the MBZIRC wall challenge (T650 with 2x bluefox down, realsense down pitched, rangefinder down)', [t650]]
fire_challenge: [False, 'Configuration for the MBZIRC fire challenge (T650 with bluefox down, realsense forward, rp lidar, 3x thermal camera, water gun)', [t650]]
fire_challenge_blanket: [False, 'Configuration for the MBZIRC fire challenge (blanket dropping) (T650 with bluefox down, realsense forward, rp lidar, 2x thermal camera, maybe blanket dropper)', [t650]]
gps_indoor_jamming: [False, 'Enable GPS jamming when drone is indoors (NOTE: UNUSED PARAMETER!)', [f450, f550, t650]]
enable_uv_leds: [False, 'Add UV LEDs to the vehicle (NOTE: T650 CONFIGURATION SEEMS WRONG!)', [f450, f550, t650]]
uvled_fr_l: [None, 'Specify UV LED frequency left', [f450, f550, t650]]
uvled_fr_r: [None, 'Specify UV LED frequency right', [f450, f550, t650]]
enable_uv_leds_beacon: [False, 'Add UV LED beacon to the top of the vehicle', [f450]]
uvled_beacon_f: [None, 'The frequency of blinking of the UV Beacon', [f450]]
enable_uv_camera: [False, 'Add UV camera on the vehicle', [f450, f550, t650]]
uvcam_calib_file: [None, 'Specify UV camera calibration different than default one', [f450, f550, t650]]
uvcam_occlusions: [False, 'Enable occlusions for UVDAR simulation (NOTE: THIS REQUIRES A BEEFY COMPUTER)', [f450, f550, t650]]
pos_file: [None, 'Load positions and ids from .csv file with format: [id, x, y, z, heading] or .yaml file with format: [uav_name: \n id: (int) \n x: (float) \n y: (float) \n z: (float) \n heading: (float)]', [f450, f550, t650]]
pos: [None, 'Specify spawn position of the first vehicle in format [x, y, z, heading]. Additional vehicles will spawn in line with 2m x-offset', [f450, f550, t650]]
